src/handlers/next/saleor-webhooks/process-saleor-webhook.ts

```diff
@@ -140,7 +137,7 @@ export const processSaleorWebhook: ProcessSaleorWebhook = async <T>({
           throw new WebhookError("Missing request body", "MISSING_REQUEST_BODY");
         }
 
-        let parsedBody: unknown;
+        let parsedBody: unknown & { version?: string | null };
```

Comment:
maybe `schemaVersion`? "version" is too generic

Reply:
field on `Event` is named `version`: https://docs.saleor.io/docs/3.x/api-storefront/miscellaneous/interfaces/event#code-style-fontweight-normal-eventbversionbcodestring-